Dockerfile

```diff
@@ -9,12 +9,13 @@ RUN apk --update upgrade && \
 ADD . /go/src/code-annotation
 WORKDIR /go/src/code-annotation
 
-RUN make build && \
+RUN make build-frontend && \
+    make build-backend && \
+    make bindata && \
     make packages
 
 FROM alpine:latest
 RUN apk --no-cache add ca-certificates
 WORKDIR /root/
-COPY --from=0 /go/src/code-annotation/build ./build
 COPY --from=0 /go/src/code-annotation/bin/code-annotation .
 CMD ["./code-annotation"]
```

Comment:
The binary should be in the PATH and as an `ENTRYPOINT` and not as a `CMD`

Reply:
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#entryopoint